kuma/users/jinja2/users/ban_user_and_cleanup_summary.html

```diff
@@ -24,28 +24,39 @@
       {{ _('The following revisions were reverted:') }}
     </h2>
     <div>
-      {{ _('(Note: No revisions were actually reverted. This feature will be added in Phase 2.)') }}
+      {{ _('(Note: No revisions were actually reverted. This feature will be added in Phase 3.)') }}
     </div>
     <br />
     <div id="revisions-reverted">
       {% if revisions_reverted %}
+        <ul>
+          {% for revision in revisions_reverted %}
+          {% endfor %}
+        </ul>
+      {% else %}
+      <p>{{ _('The user did not have any revisions that were reverted.') }}</p>
+      {% endif %}
+    </div>
 
-        {% for revision in revisions_reverted %}
-          <li>
-            <a href="{{ url('wiki.revision', revision.document.slug, revision.id) }}">{{ revision.title }}</a>
```

Comment:
Instead of ``{{ url('wiki.revision', ...)``, you should use ``{{ revision.get_absolute_url() }}``, which will take care of locales as well.